assets/scss/_features.scss

```diff
@@ -0,0 +1,26 @@
+.features {
```

Comment:
That's true, there is no media queries and on mobiles it looks like this: 
![Screenshot from 2020-03-31 23-51-47](https://user-images.githubusercontent.com/25872895/78079465-799b2000-73ac-11ea-8d86-69e46e832d33.png)

please set them in column, not in the row
also from 768px (tablets) change padding to give them more space :)